templates/install-wp-tests.sh

```diff
@@ -147,7 +147,20 @@ install_db() {
 	fi
 
 	# create database
-	mysqladmin create $DB_NAME --user="$DB_USER" --password="$DB_PASS"$EXTRA
+	if [ $(mysql -e 'show databases;' | grep ^$DB_NAME$) ]
+	then
+		echo "Reinstalling will delete the existing test database ($DB_NAME)"
+		read -p 'Are you sure you want to proceed? [No/Yes]: ' DELETE_EXISTING_DB
+		shopt -s nocasematch
+		if [[ $DELETE_EXISTING_DB =~ ^(y|yes)$ ]]
+		then
+			mysqladmin drop $DB_NAME -f --user="$DB_USER" --password="$DB_PASS"$EXTRA > /dev/null 2>&1
+			mysqladmin create $DB_NAME --user="$DB_USER" --password="$DB_PASS"$EXTRA
+		else
+			echo "Leaving the existing database ($DB_NAME) in place."
+		fi
+		shopt -u nocasematch
+	fi
```

Comment:
The structure of this `if` block is not quite right. Currently, if the database does not exist, then it will not be created.  Adding an `else` block to create the database if it didn't already exist should fix this.

Reply:
(facepalm)

Fixing now